specification/web/resource-manager/Microsoft.Web/stable/2020-12-01/Global.json

```diff
@@ -0,0 +1,329 @@
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ArmOperationResponse"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets an operation in a subscription and given region": {
+            "$ref": "./examples/GetSubscriptionOperation.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/operations/{operationId}": {
+      "get": {
+        "tags": [
+          "Global"
+        ],
+        "summary": "Gets an operation in a subscription and given region",
+        "description": "Description for Gets an operation in a subscription and given region",
+        "operationId": "Global_GetSubscriptionOperationWithAsyncResponse",
+        "parameters": [
+          {
+            "name": "location",
+            "in": "path",
+            "description": "Location name",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "description": "Operation Id",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets an operation in a subscription and given region": {
+            "$ref": "./examples/GetSubscriptionOperationWithAsyncResponse.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ArmOperationResponse": {
+      "description": "ARM Operation Response",
+      "required": [
+        "status"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Id of operation",
+          "type": "string"
+        },
+        "status": {
+          "type": "string"
+        },
+        "error": {
+          "$ref": "#/definitions/ArmOperationResponseError",
+          "description": "Error if any"
+        }
+      }
+    },
+    "ArmOperationResponseError": {
```

Comment:
>ArmOperationResponseError" [](start = 5, length = 26)

Consider referencing the common type json.

Reply:
removed.